src/python/gem5/simulate/simulator.py

```diff
@@ -191,6 +191,18 @@ def __init__(
         self._last_exit_event = None
         self._exit_event_count = 0
 
+        # self._total_insts = int(self._board.get_workload()._estimated_instructions)
```

Comment:
This doesn't need to be in the simulator. You can get the workload id in the simulator, then we can get information from there.